libevmasm/RuleList.h

```diff
@@ -146,6 +146,11 @@ std::vector<SimplificationRule<Pattern>> simplificationRuleListPart2(
 		{{Instruction::SHR, {0, X}}, [=]{ return X; }, false},
 		{{Instruction::SHL, {X, 0}}, [=]{ return u256(0); }, true},
 		{{Instruction::SHR, {X, 0}}, [=]{ return u256(0); }, true},
+		{{Instruction::LT, {X, 0}}, [=]{ return u256(0); }, true},
+		{{Instruction::GT, {X, 0}}, [=]() -> Pattern { return {Instruction::ISZERO, {{Instruction::ISZERO, {X}}}}; }, false},
```

Comment:
What about `gt(0, X)`?

Reply:
I think that should be fine.

Reply:
By which rule?

Reply:
Ah you added it! (Or I didn't see it before)